### DISCUSSION TITLE
UserSelection configuration

### DISCUSSION THREAD

**Discussion by Alireza29675** [xyflow/xyflow#2398] [General]
Hi, React Flow team 👋

I wonder if you have any plans to specify how we can configure `Selection` related settings. For example, it would be nice to decide whether we want our nodes to be selected by full-selection-overlap or partially-selection-overlap.

Or, for example, the style of `<UserSelection />` and `<NodeSelection />` component. Or various options which come with selection itself.

**Comment by moklick** [comment]
Hi @Alireza29675 !

Selection overlap `fully` and `partially` is a good idea! 

The style can be adjusted with CSS. Doesn't that work for you?
